### PR TITLE
Cosmos JS integration

### PR DESCRIPTION
**Pull request by ghost**
## Expectations

- [ ] **Test coverage: The changes of this PR are covered by test.** Unit test were added with mocks when depending on a backend/device.
- [ ] **No impact: The changes of this PR have ZERO impact on the userland.** Meaning, we can use these changes without modifying LLD/LLM at all. It will be a "noop" and the maintainers will be able to bump it without changing anything.

### CODE REVIEW

File: src/env.ts [hunk 1]
```diff
@@ -156,7 +156,7 @@ const envDefinitions = {
     desc: "location of the compound API",
   },
   COSMOS_GAS_AMPLIFIER: {
-    def: 1.4,
+    def: 4,
```
Comment (haammar-ledger) [inline src/env.ts:159]: Is it wanted to revert this value? Why?

**Review by ghost — COMMENTED** [review]
Looks good, just one question.

File: src/env.ts (hunk 1 above)
Comment (wa-aal) [reply src/env.ts]: Fix in 92f4d599d10d0f6dad8c52c0035104a4a296159a.
This is a workarround for mismatched fees estimation calculation (simulation and broadcast are on different endpoints)

**Comment by gre** [comment]
ok let's let the bot runs with this 🙏

**Pull request merged**